packages/itwinui-css/backstop/scenarios/tag.js

```diff
@@ -9,9 +9,6 @@ module.exports = [
   scenario('Type Default', {
     selectors: ['#demo-default'],
   }),
-  scenario('Type Basic', {
```

Comment:
Instead of removing the basic test, can we change it so that the basic demo does not have truncation? Then we can add a separate truncation demo that we can exclude from tests.

Reply:
Removed truncation from the basic test and added new one with truncation.